swan-lake/development-tutorials/source-code-dependencies/package-references.md

```diff
@@ -337,7 +337,19 @@ warning: Detected conflicting jar files:
 
 **Define the scope for a dependency**
 
-By default, the scope takes the value `default` which will add it to the final executable JAR file. If you want to restrict a certain platform dependency to be used only for testing, specify the scope as `testOnly`. This will add the platform dependncy to the test runtime but will avoid packing it into the final executable JAR file.
+By default, when the scope has not been explicitly specified for a platform dependency in the `Ballerina.toml`, it will be packaged into the final executable JAR file or the BALA file. There are two scopes that can be used to restrict this behaviour.
+
+***'testOnly' scope***
+
+To restrict a certain platform dependency to be used only for testing, specify the scope as `testOnly`. This will add the platform dependency to the test runtime but will avoid packing it into the final executable JAR file.
```

Comment:
```suggestion
To restrict a certain platform dependency from being used only for testing, specify the scope as `testOnly`. This will add the platform dependency to the test runtime but will avoid packing it into the final executable JAR file.
```

Reply:
Did not add this change since it gives the idea that the scope will restrict the jar from being used only for testing instead of allowing the jar to be used only for testing, which is the actual purpose of the scope.